assets/css/app.scss

```diff
@@ -8,6 +8,8 @@
 @import "refills/flashes";
 @import "refills/breadcrumb";
 @import "refills/flex_box";
+@import "refills/badges";
```

Comment:
Prefer single quoted strings

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#stringquotes)